### DISCUSSION TITLE
ARM Overwrites Unknown Album

### DISCUSSION THREAD

**Discussion by trleith** [automatic-ripping-machine/automatic-ripping-machine#971] [General]
When you rip a CD that ARM can't match to anything, it creates Unknown Artist ==> Unknown Album

If you rip another one like that, it overwrites the tracks (i.e. 01 - Track 01.flac) from the first rip in the (one and only) Unknown Album folder.

**Proposed Solution**

ARM should serialize the unknown albums -- if there's an existing folder "Unknown Album" create a new folder "Unknown Album 01" "Unknown Album 02" ... "Unknown Album nn" to stop these collisions.

**Comment by 1337-server** [comment]
Known issue: https://github.com/automatic-ripping-machine/automatic-ripping-machine/issues/929
Fix: https://github.com/automatic-ripping-machine/automatic-ripping-machine/pull/980

**Comment by pwarren** [comment]
Still doing it to me, with ARM 2.6.75, installed via docker container on a debian box. 
subsequent CDs with no entries in the default lookups overwrite the previous one in /home/arm/music/Unkown Artist/Unkown Album/

For me it's some super obscure CDs from late 90s Canberra bands.

**Reply by microtechno9000** [reply]
Can you raise an issue for linking to a fix, if you have a log please add

**Reply by pwarren** [reply]
Sure thing #1142 created, thanks!
